src/main/java/org/jenkinsci/plugins/vSphereCloudSlaveTemplate.java

```diff
@@ -507,7 +507,6 @@ private ComputerLauncher determineLauncher(final VSphere vSphere, final String c
                     sshLauncher.getPrefixStartSlaveCmd(), sshLauncher.getSuffixStartSlaveCmd(),
                     sshLauncher.getLaunchTimeoutSeconds(), sshLauncher.getMaxNumRetries(),
                     sshLauncher.getRetryWaitTime(), sshLauncher.getSshHostKeyVerificationStrategy());
-            return launcherWithIPAddress;
```

Comment:
I must've missed this first time around, but this doesn't look right to me - that'll mean nobody can use a SSHLauncher to connect the slave to Jenkins.

Reply:
not sure what happened here, must be one of my many merge conflicts that has gone bad.

Reply:
ah, revert cleanup failed to cleanup this merge conflict 😆

Reply:
Cleaned up the git history

Reply:
Sorry for the confusion, I confused myself 😭